tests/utils.py

```diff
@@ -117,7 +117,7 @@ def runAsPython(test_dir, main_code, extra_code=None, run_in_function=False, arg
         args = []
 
     proc = subprocess.Popen(
-        ["python", "test.py"] + args,
+        ["python3", "test.py"] + args,
```

Comment:
Agreed that this is a good safety mechanism; however, it would probably be better to use `sys.executable`, which will ensure that the subprocess uses *exactly* the same Python interpreter as the test environment, and allow for a couple of odd situations where the executable won't be called `python`.